packages/flutter/lib/src/widgets/scrollable.dart

```diff
@@ -251,6 +251,19 @@ class Scrollable extends StatefulWidget {
     return widget?.scrollable;
   }
 
+  /// Gets the current scroll velocity of the state of the closest instance of
+  /// this class that encloses a given context.
+  ///
+  /// This does _not_ cause callers to subscribe to updates of this widget, and
+  /// is only safe to use as a single point in time reference.
+  ///
+  /// This value is in logical pixels per second. It may be positive, negative,
+  /// 0, or null.
```

Comment:
Document what null means?

(here it's correct what you said above: You're not in a scrollable context)